raiden/tests/unit/test_binary_encoding.py

```diff
@@ -106,7 +86,7 @@ def test_mediated_transfer_min_max(amount, payment_identifier, fee, nonce, trans
             fee=fee,
         )
     )
-    assert decode(mediated_transfer.encode()) == mediated_transfer
+    mediated_transfer.packed()
```

Comment:
Seems to have no effect, missing assert?

Reply:
I didn't come up with a good assert, so I thought I could at least do a smoke test. Any suggestion for a good assert?

Reply:
What is this test even testing? That the entire range of values can be represented in the message? Seems a bit like testing the language...

Reply:
Things like `AttributeError`s or failing type conversions while packing. If everything is properly typed, mypy should be able to catch those errors, though. I have no problems removing this, if that's preferred.

Reply:
Yeah sounds good. Maybe add a comment that it’s intentional there is no assert.